book/02-git-basics/sections/recording-changes.asc

```diff
@@ -268,15 +268,15 @@ Está fora do escopo deste livro explicar os detalhes de múltiplos arquivos `.g
 ====
 
 [[r_git_diff_staged]]
-==== Viewing Your Staged and Unstaged Changes
+==== Visualizando suas Alterações Preparadas e Não-Preparadas
```

Comment:
Não sei se traduzir _staged_ e _unstaged_ como _preparadas_ e _não-preparadas_ é uma boa ideia. Veja que, ao longo do texto, eu acabei usando expressões como "mandar para o _stage_", em vez de _preparar_. Acho que o título não ficou coerente com o texto. Também estou achando que a tradução _preparada_ para _staged_ não é uma boa opção. Os desenvolvedores de software costumam usar bastante os estrangeirismos. Acho que forçar a tradução deixaria o texto estranho para eles. 

Tem alguma recomendação para esses casos?

Reply:
É um dilema realmente. Eu desconheço se existe uma tradução consagrada para esses termos. O ideal seria definirmos um guia de estilo para a tradução, onde essas questões fossem padronizadas.

No caso do título, eu tenho uma sugestão: "Visualizando Suas Alterações Dentro e Fora do Stage". O que você acha?

> Veja que, ao longo do texto, eu acabei usando expressões como "mandar para o _stage_", em vez de _preparar_.

Legal, acho que fica melhor mesmo.

Reply:
Acho que assim – _"(...) Dentro e Fora do Stage"_ – fica bom! Já fiz essa alteração.

Ter um guia de estilos seria uma boa! Vi [esse exemplo aqui](https://translations.ted.com/Guia_de_Estilo_para_Tradu%C3%A7%C3%B5es_e_Transcri%C3%A7%C3%B5es_em_Portugu%C3%AAs_(Brasil)_no_TED_Translators) do TED. Eles tratam de vários problemas comuns, além de fazer recomendações. Vou ver se rascunho alguma coisa parecida.